README.md

```diff
@@ -44,6 +44,11 @@ https://cdnjs.com/libraries/jquery.mask
 ### JSDelivr
 http://www.jsdelivr.com/projects/jquery.mask
 
+## RubyGems
+```ruby
+gem 'jquery_mask_rails', '~> 0.1.0'
```

Comment:
@maurcarvalho Would be better if remove the version. As you upgrade the gem this would be outdated.